cmd/calico-node/main.go

```diff
@@ -44,6 +45,9 @@ var birdReady = flagSet.Bool("bird-ready", false, "Run BIRD readiness checks")
 var bird6Ready = flagSet.Bool("bird6-ready", false, "Run BIRD6 readiness checks")
 var felixReady = flagSet.Bool("felix-ready", false, "Run felix readiness checks")
 
+// thresholdTime is introduced for bird readiness check. Default value is 30 sec.
+var thresholdTime = flagSet.Duration("thresholdTime", 30*time.Second, "Threshold time for bird readiness")
```

Comment:
For consistency with other params above, `"thresholdTime"` should be `"threshhold-time"`

Reply:
Acknowledged and updated.